setup.py

```diff
@@ -1,61 +1,3 @@
 from setuptools import setup
 
-# we pin these dependencies in the requirements files -- all of these
-# should be python 3 compatible
-DEPENDENCIES = [
-    "glean_sdk>=60.3.0",
-    "beautifulsoup4>=4.7.1",
-    "colorama>=0.4.1",
-    "configobj>=5.0.6",
-    "distro>=1.8.0",
-    "importlib_resources>= 5.10",
-    "mozdevice>=4.1.0,<5",
-    "mozfile>=2.0.0",
-    "mozinfo>=1.1.0",
-    "mozinstall>=2.0.0",
-    "mozlog>=4.0",
-    "mozprocess>=1.3.1",
-    "mozprofile>=2.2.0",
-    "mozrunner>=8.0.2",
-    "mozversion>=2.1.0",
-    "redo>=2.0.2",
-    "requests>=2.21.0",
-    "taskcluster>=6.0.0",
-]
-
-desc = """Regression range finder for Mozilla nightly builds"""
-long_desc = """Regression range finder for Mozilla nightly builds.
-For more information see the mozregression website:
-http://mozilla.github.io/mozregression/"""
```

Comment:
Looks like this was dropped? It could result in more blankness on the main PyPI page.

Reply:
pyproject somehow does not support this anymore and accepts readme instead 👀 (line 28)